addons/mail/static/src/discuss/call/common/call_action_list.xml

```diff
@@ -88,6 +88,15 @@
                     </div>
                 </button>
             </div>
+            <div t-if="isSmall and store.settings.use_push_to_talk and isOfActiveCall" class="d-flex align-items-center flex-wrap justify-content-between p-2">
+                <button class="o-discuss-CallActionList-pushToTalk btn btn-primary d-flex w-100 border-0 shadow-none"
+                    aria-label="Push to talk"
+                    title="Push to talk"
+                    t-on-touchstart="onTouchstartPushToTalk"
+                    t-on-touchend="onTouchendPushToTalk">
```

Comment:
Prefer to make it work also with mouse click, otherwise this is not great for testing.

Also the different visuals of active / inactive should not rely on touch-holding specific style of the button, so that:
- it also works with long click press
- there is no latency between card PTT outline and the button state (both should be in sync)

Reply:
Actually latency is not that bad. But the touch-press effect is distinct enough in light theme but not in dark theme.
So another reason not plainly relying on it but have explicit style in sync with PTT on/off

Reply:
I think it might not be worth adding the mouse click. Now, testing in the browser is accepted and adding more LOC is harder to maintain as well, especially when the code is not used in real proc environment.

Reply:
Ok didn't know touch events were superset of mouse events, so `touchstart` includes `mousedown`. Then that's good :)